python/pyspark/mllib/clustering.py

```diff
@@ -49,6 +49,9 @@ class KMeansModel(Saveable, Loader):
 
     """A clustering model derived from the k-means method.
 
+    :param centers:
+      Initial cluster centers
```

Comment:
Maybe I should just remove this change, the model is not really meant to be created manually anyway.